ibis/backends/base_sql/__init__.py

```diff
@@ -11,15 +11,14 @@
 from typing import Optional
 
 import ibis
-import ibis.backends.base_sqlalchemy.compiler as comp
 import ibis.common.exceptions as com
 import ibis.expr.analysis as L
 import ibis.expr.datatypes as dt
 import ibis.expr.operations as ops
 import ibis.expr.types as ir
 import ibis.util as util
 from ibis.backends.base_sqlalchemy import transforms
-from ibis.impala import identifiers
+from ibis.backends.impala import identifiers
```

Comment:
We shouldn't have imports from the base backends to the specific ones, only the other way round. Can you move this `identifiers` to the base backend please?